configure

```diff
@@ -3730,6 +3730,7 @@ sr_filter_select="dnn"
 stereo3d_filter_deps="gpl"
 splitcc_filter_deps="avcodec"
 subscale_filter_deps="swscale avcodec"
+text2bitmap_filter_deps="libass"
```

Comment:
Needs to be added in alphabetical order.
But it seems I'll need to fix my own entries as well. The frequent rebasing has mixed up the order.